model/shared/src/main/scala/explore/model/Observation.scala

```diff
@@ -179,33 +183,76 @@ case class Observation(
   lazy val constraintsSummary: String =
     s"${constraints.imageQuality.label} ${constraints.cloudExtinction.label} ${constraints.skyBackground.label} ${constraints.waterVapor.label}"
 
+  private val executedStates =
```

Comment:
`ExecutedStates`? Would it make sense to put it in `Constants`, or will it only be used here?

Reply:
As far as I know, it's only needed here. I guess that could change in the future?

Reply:
Ok, anyway we use CamelCase for constants, it's in the new code guidelines ;)

Reply:
We have guidelines? 😮 Anyway, their just guidelines - kind of like the Pirates Code. 😆